code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -53,7 +53,6 @@ proc/issyndicate(mob/living/M as mob)
 	for(var/datum/mind/synd_mind in syndicates)
 		synd_mind.assigned_role = SPECIAL_ROLE_NUKEOPS //So they aren't chosen for other jobs.
 		synd_mind.special_role = SPECIAL_ROLE_NUKEOPS
```

Comment:
Wouldnt it make sense to also change these too?

Reply:
I don't *think* so.
mind.offstation_role isn't overwritten at roundstart, and thus persists into normal rounds, causing bugs. It has to be moved to fix them.
mind.assigned_role and mind.special_role DO get overwritten. So I don't think they need to be moved. I'm also concerned that moving them will potentially introduce other bugs.

Frankly, at some point we should probably rewrite all game_mode can_start procs to avoid setting ANYTHING that could potentially persist... but that's ridiculously out of scope for this PR. This PR is just about changing the thing which is definitely causing issues.

Reply:
Tested locally and both of these are null at roundstart, but that was with a small sample size on a local server, so its unsure how it would react on live.